### ISSUE TITLE
[Bug]: Deprecation errors after updating to 4.26.1

### ISSUE THREAD

**Issue by deluxetom** [nelmio/NelmioApiDocBundle#2279]
### Version

4.26.1

### Description

I'm seeing a ton of these messages now:
```
deprecation.INFO: User Deprecated: Since zircote/swagger-php 4.9: Context detecting is deprecated {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since zircote/swagger-php 4.9: Context detecting is deprecated at /app/vendor/zircote/swagger-php/src/Context.php:197)"} []
```

### Additional context

Related ticket: https://github.com/zircote/swagger-php/issues/1579

I'm not sure how we can fix that with `NelmioApiDocBundle` or it's an issue with `swagger-php`

**Comment by DerManoMann** [comment]
It's something I've tried before in swagger-php and I thought things got fixed here, but apparently not :/

Maybe I need a completely different approach - sigh.

**Comment by D3strukt0r** [comment]
For more context for everyone, came from this merge: https://github.com/zircote/swagger-php/pull/1563

**Comment by DerManoMann** [comment]
This should be fixed in swagger-php 4.9.2 -  https://github.com/zircote/swagger-php/pull/1586

**Comment by deluxetom** [comment]
@DerManoMann thank you!
